tests/Microsoft.Identity.Test.Unit/CoreTests/MsalIdHelperTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.PlatformsCommon.Shared;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Unit.CoreTests
+{
+    [TestClass]
+    public class MsalIdHelperTests
+    {
+        [TestMethod]        
+        public void XClientOSTest()
+        {
+            // Act
+            IDictionary<string, string> parameters = MsalIdHelper.GetMsalIdParameters(null);
+
+            // Assert
+            Assert.IsTrue(parameters.ContainsKey(MsalIdParameter.OS));
```

Comment:
>OS [](http://example.com/codeflow?start=65&length=2)

if the goal is to synchronize with our xplat library here are the values MSAL.CPP sends today.

x-client-SKU: MSAL.xplat.Win32
x-client-Ver: 1.1.0+ebfe9dc1
x-client-src-SKU: MSAL.xplat.Win32
x-client-OS: 10.0.19041.3636